### PR TITLE
Media Editing Basics Phase One - Master Issue

### PR DESCRIPTION
**Pull request by ashiagr**
Master Thread: paCBwp-jY-p2
Project Kickoff: p8g8V2-dR-p2
Design: paCBwp-ll-p2

**Phase 1**
Edit an already uploaded image
- [x] Introduce a new module for media editing
- [x] Introduce media editing intermediary screen - the Hub
- [x] Link the module to the main project
- [x] Show low resolution image on the Hub 
- [x] Show loading indicator and blur effect on the Hub 
- [x] Load the high res version of the image into a file
- [x] Start uCrop with the high resolution version of the image
- [x] Replace the original image with the edited image in Gutenberg editor
- [x] Enable "Edit Image" button in Gutenberg-mobile for Android
- [x] Track usage

All the changes in this PR have already been tested. Just smoke test the app before merging the changes into develop.

P.S.
Created a draft PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/2062 to enable media editing for Android in `gutenberg-mobile`. Feature will be available once it is merged to develop. Till that time we might want to update the commit hash from the draft PR in `WPAndroid` for the purpose of testing. I'm not aware how we do it.

### CODE REVIEW

File: libs/image-editor/ImageEditor/src/androidTest/kotlin/org/wordpress/android/imageeditor/ExampleInstrumentedTest.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import org.junit.Assert.assertEquals
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
```
Comment (malinajirka) [inline libs/image-editor/ImageEditor/src/androidTest/kotlin/org/wordpress/android/imageeditor/ExampleInstrumentedTest.kt]: I think we might want to remove this class, wdyt?

File: build.gradle [hunk 2]
```diff
@@ -30,6 +33,7 @@ allprojects {
         google()
         jcenter()
         maven { url "https://dl.bintray.com/wordpress-mobile/maven" }
+        maven { url "https://jitpack.io" }
```
Comment (malinajirka) [inline build.gradle]: Could you please double check if we really need this. Thanks!

**Review by malinajirka — APPROVED** [review]
Thanks @ashiagr!

I've left two minor comments. Everything else looks good to me! Feel free to merge it.

File: libs/image-editor/ImageEditor/src/androidTest/kotlin/org/wordpress/android/imageeditor/ExampleInstrumentedTest.kt (hunk 1 above)
Comment (ashiagr) [reply libs/image-editor/ImageEditor/src/androidTest/kotlin/org/wordpress/android/imageeditor/ExampleInstrumentedTest.kt]: Removed 3b2749e.

File: build.gradle (hunk 2 above)
Comment (ashiagr) [reply build.gradle]: It wasn't needed. Removed 3b2749e.

**Pull request merged**